src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -134,6 +139,14 @@ static void set_intersection(const std::set<std::vector<element::Type>>& precisi
 InferenceEngine::Precision eltwise_precision_helper::get_precision(const size_t inputs_number,
                                                                    const InferenceEngine::Precision(&src_prc)[MAX_ELTWISE_INPUTS],
                                                                    const std::vector<Eltwise::EltwiseData>& eltwise_data) {
+    // TODO: refactor
```

Comment:
Are you going to refactor that in this PR? Please use `one_of()` at least.

Reply:
done, thanks